Tasks/Common/azurerest-common/npm-shrinkwrap.json

```diff
@@ -0,0 +1,238 @@
+{
```

Comment:
Can you match the file count and file versions with the OSS third party notice file that we are preparing.